README.md

```diff
@@ -4,9 +4,33 @@
 
 Welcome to the EOS.IO source code repository!
 
+# Table of contents
+
+1. [Getting Started](#gettingstarted)
+2. [Setting up a buid/development environment](#setup)
+	1. [Clean install Ubuntu 16.10](#ubuntu)
+	2. [macOS Sierra 10.12.6](#macos)
+3. [Building and running a node](#runanode)
```

Comment:
Perhaps this section should be titled "Building EOS and running a node"

Reply:
Tweaked accordingly